src/frontend/src/pages/HomePage/components/InfoBlock.tsx

```diff
@@ -28,13 +33,33 @@ const InfoBlock: React.FC = () => {
       <Grid item>
         <Box
           sx={{
-            backgroundColor: 'gray',
+            backgroundColor: theme.palette.background.paper,
             height: '25vh',
             borderRadius: '10px',
             width: '100%'
           }}
         >
-          <Typography variant="h5">Useful Links</Typography>
+          <Typography variant="h5" sx={{ mb: 3, px: 2, pt: 2 }}>
+            Useful Links
+          </Typography>
+
+          <Grid container spacing={2} justifyContent="center" sx={{ px: 2 }}>
+            {links.map((link) => {
+              return (
+                <Grid item xs={6}>
+                  <Button
+                    variant="contained"
+                    fullWidth
+                    sx={{ backgroundColor: '#616161', color: 'white' }}
```

Comment:
Can you add a border radius of like 10px to make it more rounded

Reply:
You can also make the links a little thicker by adding this prop as well: height: '5vh'